src/main/java/de/adorsys/keycloak/config/repository/ExecutionFlowRepository.java

```diff
@@ -86,7 +87,7 @@ public void createExecutionFlow(
         logger.trace("Create non-top-level-flow in realm '{}' and top-level-flow '{}'", realmName, topLevelFlowAlias);
 
         AuthenticationManagementResource flowsResource = authenticationFlowRepository.getFlowResources(realmName);
-        flowsResource.addExecutionFlow(topLevelFlowAlias, executionFlowData);
+        flowsResource.addExecutionFlow(topLevelFlowAlias, new HashMap<>(executionFlowData));
```

Comment:
Is the order of the executionFlowData relevant here? If it is, then a LinkedHashMap that preserves the order would be more suitable here.

Reply:
Hi @thomasdarimont, as the source type is also HashMap, I suggest that the order is not important ;)